actionpack/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Allow hosts redirects from `hosts` Rails configuration
```

Comment:
I think providing an example in the changelog would be nice

Reply:
What example would you like to view here?
This PR only fixes the raise of `UnsafeRedirectError` when redirecting to an allowed hosts from `config.hosts`.